app/ts/ManualInput/ManualInputTable/CriteriaRows/CriterionRow/DataSourceRow/InputCell/EffectCell/EffectCell.tsx

```diff
@@ -19,10 +18,15 @@ export default function EffectCell({alternativeId}: {alternativeId: string}) {
 
   const effect = getEffect(criterion.id, dataSource.id, alternativeId);
 
-  const INVALID_VALUE = 'Invalid value';
-
   useEffect(() => {
-    setLabel(createLabel(effect));
+    setLabel(
```

Comment:
During manual input, the label 'Empty' was displayed and it was clickable. Now, there is nothing to click.